### ISSUE TITLE
Label flash when changing tab

### ISSUE THREAD

**Issue by RTsosssoldi** [lukepighetti/salomon_bottom_bar#26]
Hi, really good project!
Could someone tell me why when navigating to the left, the label flash for some milliseconds? 
I attached a small video where u can see what i'm talking about.

https://imgur.com/a/cBRvUkP

**Comment by lukepighetti** [comment]
I can't reproduce this on `3.2.0` with the [example project](https://github.com/lukepighetti/salomon_bottom_bar/tree/main/example), can you let me know if it's still present?

https://user-images.githubusercontent.com/666539/159277261-afc5f7ea-ffb4-4dc2-b37d-6c75ae9002fb.mov

**Comment by RTsosssoldi** [comment]
Hey, sorry if i'm late! @lukepighetti 
Yes it's still present, the video i posted was made by recording a One Plus 6 on Android 11.
salomon_bottom_bar v3.2.0

**Comment by RTsosssoldi** [comment]
Hey, sorry if i'm late! @lukepighetti
If i try the example project, i can't see any problem, so i tried to update to the 3.3.0 on my project but the problem still.

Device: One Plus 6 on Android 11.

**Comment by lukepighetti** [comment]
Can you put together a [MCVE](https://stackoverflow.com/help/minimal-reproducible-example)?

**Comment by RTsosssoldi** [comment]
Well, i'll try in the next hours!

**Comment by barbrady91** [comment]
Hello, I have the same problem.

To reproduce the problem, all you have to do is create a new Flutter sample project and add the following source code to the scaffold:

   ```
      bottomNavigationBar: SalomonBottomBar(
        onTap: (index) {
          setState(() {
            _counter = index;
          });
        },
        currentIndex: _counter,
        selectedItemColor: Colors.blue,
        items: [
          SalomonBottomBarItem(
            icon: const Icon(Icons.home, color: Colors.black),
            title: const Text('Dashboard', style: TextStyle(color: Colors.black)),
          ),
          SalomonBottomBarItem(
            icon: const Icon(Icons.calendar_month, color: Colors.black),
            title: const Text('Planer', style: TextStyle(color: Colors.black)),
          ),
          SalomonBottomBarItem(
            icon: const Icon(Icons.task, color: Colors.black),
            title: const Text('Aufgaben', style: TextStyle(color: Colors.black)),
          ),
```

To make this as easy as possible, I used the _counter as the index.

Of course, the dependency must still be added to the pubspec.yaml.
I used the current latest version ^3.3.0.

I have tested in the iPhone 13 simulator, on a real iPhone 7 and on a real Samsung Galaxy device. In all 3 cases I had the same problem.

I also uploaded my test project to GitHub: https://github.com/barbrady91/salomon_issue

I would be very happy if you could reply.

Greetings Marc

![issue](https://user-images.githubusercontent.com/49312977/167260975-9833f66a-10f9-42b7-80b7-8d0a865a7dec.gif)

**Comment by lukepighetti** [comment]
Thank you for the MCVE, this has been resolved in `3.3.1` which is now on pub

**Comment by barbrady91** [comment]
I just updated to version 3.3.1. Now everything works. Thank you very much for fixing the issue so quickly.
And thank you for the great package in general. Because of the issue, I was looking for an alternative, but couldn't find one that I like as much. The SalomonBottomBar just looks fantastic. Please keep up the great work.
